content/language-agnostic/encyclopedia/code-editor.md

```diff
@@ -0,0 +1,18 @@
+<!--
+---
+Subjects:
+  - "What is a Code Editor?"
+Tags: 
+  - "language-agnostic"
+  - "code editor"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
```

Comment:
We're thinking we'll just link to course/path landing pages for now, since /articles content doesn't show off all that Codecademy has to offer, and we're focusing on creating long-term engagement. This is mentioned in the content standards, but I can add it to the templates, since I think info provided there will be much easier to remember.

Reply:
Sonny says he'd link to Code Foundations or Python 3, for what it's worth. This is a relatively introductory entry, so I think Code Foundations might be my first choice.